prusti-viper/src/encoder/errors/error_manager.rs

```diff
@@ -52,6 +52,8 @@ pub enum ErrorCtxt {
     /// A Viper `assert false` that encodes the failure (panic) of an `assert` Rust terminator
     /// Arguments: the message of the Rust assertion
     AssertTerminator(String),
+    /// A Viper `assert false` in the context of a bounds check (if we were able to detect it)
```

Comment:
> "if we were able to detect it"

Do you know of any cases when the bounds check is not detected? If not, let's be more positive in the comment and leave out the last part.

Reply:
hm, you're right, i don't know of any cases we're missing, as long as rustc has the correct `AssertKind` on them.. removed the last part.